### PR TITLE
EWL-7222: Evergreen Page | Add page title and subtitle to SG

### PR DESCRIPTION
**Pull request by caharder**
## Ticket(s)

**Jira Ticket**
- [EWL-7222: Evergreen Page | Add page title and subtitle to SG](https://issues.ama-assn.org/browse/EWL-7222)

## Description
Added a variation of the masthead organism that centers the title and subtitle

## To Test
- [ ] Pull branch /feature/EWL-7222-centered-masthead
- [ ] run gulp serve
- [ ] visit [http://localhost:3000/?p=organisms-masthead-with-subtitle-as-centered](http://localhost:3000/?p=organisms-masthead-with-subtitle-as-centered)
- [ ] Verify that new masthead variation has centered text as seen in screenshot below
- [ ] Verify that new masthead variation category is formatted as seen in screenshot below
- [ ] Verify that new masthead variation social icons are formatted as seen in screenshot below (except for color of icons, disregard the purple color in the screenshot)

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
Screen Shot 2019-05-29 at 12 12 28 PM

## Remaining Tasks
Add class to D8 page template for evergreen pages - [https://github.com/AmericanMedicalAssociation/ama-d8/pull/1415](https://github.com/AmericanMedicalAssociation/ama-d8/pull/1415)

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/03-organisms/_masthead.scss [hunk 1]
```diff
@@ -40,6 +40,12 @@
 
     &__title {
       width: 100%;
+
+      &--centered {
+        p, h1, h2, h3, h4, h5, h6 {
```
Comment (designerneil) [inline styleguide/source/assets/scss/03-organisms/_masthead.scss:49]: perhaps a wildcard would work better? *

File: styleguide/source/_patterns/03-organisms/masthead/masthead~with-subtitle-as-centered.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      "href": "#",
+      "text": "Delivering care",
+      "target": "_self",
+      "class": "ama__masthead__content__link"
+    },
+    "heading": {
+      "text": "JAMA Network to launch fully open-access journal"
+    },
+    "subtitle": {
+      "text": "Example text for a subtitle, lorem ipsum sit dolor"
```
Comment (designerneil) [inline styleguide/source/_patterns/03-organisms/masthead/masthead~with-subtitle-as-centered.json]: Let's match the text here to the Zeplin

Text describing what this rule means and how the AMA has provided the below to help. Lorem Ipsum. Alsip Clark Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam et felis vel massa consequat congue. Praesent viverra feugiat arcu aliquam posuere. Integer non luctus urna, quis tincidunt nisi. Proin ullamcorper velit in vulputate sodales. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Cras tincidunt id nisi ac ornare. Nam vel fermentum ligula.

**Review by designerneil — CHANGES_REQUESTED** [review]
font size

File: styleguide/source/assets/scss/03-organisms/_masthead.scss (hunk 1 above)
Comment (caharder) [reply styleguide/source/assets/scss/03-organisms/_masthead.scss:49]: I don't think you can put a wildcard on just headers and I do not want to target everything inside --centered.

File: styleguide/source/_patterns/03-organisms/masthead/masthead~with-subtitle-as-centered.json (hunk 2 above)
Comment (caharder) [reply styleguide/source/_patterns/03-organisms/masthead/masthead~with-subtitle-as-centered.json]: Added Zeplin text and fixed the font size and weight of the subtitle.

**Review by designerneil — APPROVED** [review]

**Pull request merged**